### PR TITLE
update description of rhel9 ospp profile

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- uipdate description in the profile file

#### Rationale:

aligning with purpose of the profile

### CODE REVIEW

**Review by ggbecker — CHANGES_REQUESTED** [review]
Can you break the text into multiple lines?

File: products/rhel9/profiles/ospp.profile [hunk 1]
```diff
@@ -11,14 +11,8 @@ reference: https://www.niap-ccevs.org/Profile/PP.cfm
...
 description: |-
-    This profile reflects mandatory configuration controls identified in the
-    NIAP Configuration Annex to the Protection Profile for General Purpose
-    Operating Systems (Protection Profile Version 4.2.1).
-
-    This configuration profile is consistent with CNSSI-1253, which requires
-    U.S. National Security Systems to adhere to certain configuration
-    parameters. Accordingly, this configuration profile is suitable for
-    use in U.S. National Security Systems.
+    This profile is part of Red Hat Enterprise Linux 9 Common Criteria Guidance documentation for Target of Evaluation based on Protection Profile for General Purpose Operating Systems (OSPP) version 4.2.1 and Functional Package for SSH version 1.0.
```
Comment (ggbecker) [inline products/rhel9/profiles/ospp.profile:14]: I wonder where this OpenSSH 1.0 version comes from.

Comment (adelton) [reply products/rhel9/profiles/ospp.profile:14]: It's not OpenSSH.

It's Functional Package for SSH in https://www.niap-ccevs.org/Profile/Info.cfm?PPID=459&id=459 which replaced the older Extended Package (https://www.niap-ccevs.org/Profile/Info.cfm?PPID=389&id=389).

**Comment by vojtapolasek** [comment]
I have fixed the line length.

File: products/rhel9/profiles/ospp.profile (hunk 1 above)
Comment (ggbecker) [reply products/rhel9/profiles/ospp.profile]: Thanks for the explanation.

**Review by ggbecker — APPROVED** [review]
LGTM. Thanks for addressing the request.

File: products/rhel9/profiles/ospp.profile [hunk 2]
```diff
@@ -11,14 +11,12 @@ reference: https://www.niap-ccevs.org/Profile/PP.cfm
...
-
-    This configuration profile is consistent with CNSSI-1253, which requires
-    U.S. National Security Systems to adhere to certain configuration
-    parameters. Accordingly, this configuration profile is suitable for
-    use in U.S. National Security Systems.
+    This profile is part of Red Hat Enterprise Linux 9 Common Criteria Guidance
+    documentation for Target of Evaluation based on Protection Profile for
+    General Purpose Operating Systems (OSPP) version 4.2.1 and Functional
+    Package for SSH version 1.0.
+    Where appropriate, CNSSI 1253 or DoD-specific values are used for
```
Comment (ggbecker) [inline products/rhel9/profiles/ospp.profile:18]:
```suggestion
    Package for SSH version 1.0.

    Where appropriate, CNSSI 1253 or DoD-specific values are used for
```

**Pull request merged**